README.md

```diff
@@ -901,6 +901,34 @@ type TotallyMutableFoo = Mutable<Foo>
 
 `Readonly` is a type mapper to make all properties of an object to be readonly.
 
+## Enums
+[Official Documentation](https://www.typescriptlang.org/docs/handbook/enums.html)
+> Enums allow us to define a set of named constants. Using enums can make it easier to document intent, or create a set of distinct cases. TypeScript provides both numeric and string-based enums.
+
+Enums are more maintainable compared to literal types. You can easily find usages, and rename enum values.
```

Comment:
This statement is somewhat an opinion (I wouldn't say it sounds completely impartial).

Reply:
Please take a look now, if still fills like an opinion, please suggest or change to what you think is more subjective (or delete entirely).